pkg/server/commands/connectedobjects/connected_objects.go

```diff
@@ -152,6 +152,7 @@ const (
 
 type ConnectedObjectsResult struct {
 	Object       string
+	QueryCount   uint32
```

Comment:
It's weird to me that we are adding this count `per object` and not as a total per request

Reply:
Updated to use overall count